DependencyInjection/Configuration.php

```diff
@@ -168,7 +205,11 @@ private function addBodyListenerSection(ArrayNodeDefinition $rootNode)
                     ->fixXmlConfig('decoder', 'decoders')
                     ->addDefaultsIfNotSet()
                     ->canBeUnset()
+                    ->treatFalseLike(array('enabled' => false))
```

Comment:
I tried using ``canBeEnabled()`` here, but it didn't work as expected as it ended by defaulting ``enabled`` to ``false``, so I just added the code for ``canBeEnabled()`` for this, removed ``treatNullLike()`` and switched the default of ``enabled`` to ``true``.